README.md

```diff
@@ -5,10 +5,17 @@ Putting your settings in `settings.py` or as environment variables also mean an
 This Django app leverage the built-in Django admin so changing settings is easier as you can use the web interface.
 
 ## Requirements
-* Python 3
-* Django >= 1.11
+* Python = "^3.6"
+* Django = "^3"
 
-I only did a quick test with Python 3.6 on Django 1.11 and the latest Django, which is 2.2.6 at the time being.
+Run the following command inside project's root folder to install it's dependencies with Poetry:
```

Comment:
I have a question. Poetry is needed when we need to build this thing. But, if anyone who only wants to use it, they can just use pip, pipenv, poetry or whatever package management they want, right? Shouldn't this be more like a generic requirement for users? I think poetry should be a requirement for building it, not installing it, so maybe we should make another section for building this.

Any thoughts? @mattwang44

Reply:
What you describe is TOTALLY CORRECT! The current readme combines the manual for regular package users and the contributors and anyone not familiar with the current status could be confused with it. My thought is, after we are done implementing the unit testing and confirming how to run the test suites, we should extract those contributing guidelines into `CONTRIBUTING.md`. This should be the last step before the next release.